pkg/inject/envoy.go

```diff
@@ -61,6 +69,22 @@ const envoyContainerTemplate = `
     {
       "name": "ENABLE_ENVOY_XRAY_TRACING",
       "value": "1"
+    }{{ end }}{{ if .EnableXrayTracing }},
+    {
+      "name": "XRAY_DAEMON_PORT",
+      "value": "{{ .XrayDaemonPort }}"
```

Comment:
Since we configure the xray sidecar and inject it, this needs to be changed in xray injector as well [1]

[1] https://github.com/aws/aws-app-mesh-controller-for-k8s/blob/master/pkg/inject/xray.go#L19